src/Umbraco.Web.UI.Client/src/packages/core/components/input-number-range/input-number-range.element.ts

```diff
@@ -99,6 +102,12 @@ export class UmbInputNumberRangeElement extends UmbFormControlMixin(UmbLitElemen
 				return this._minValue !== undefined && this._maxValue !== undefined ? this._minValue > this._maxValue : false;
 			},
 		);
+
+		this.addValidator(
+			'valueMissing',
+			() => '#validation_fieldIsMandatory',
```

Comment:
To keep things consistent with the other inputs, we need to add a requiredMessage property. If it's not provided, we'll fall back to UMB_VALIDATION_EMPTY_LOCALIZATION_KEY.

Reply:
The UI Library doesn't know how to localize text, so we handle this both for consistency and, more importantly, to ensure the message is in the correct language.